src/Symfony/Component/Cache/Adapter/CouchbaseCollectionAdapter.php

```diff
@@ -221,13 +220,4 @@ protected function doSave(array $values, $lifetime)
 
         return [] === $ko ? true : $ko;
     }
-
-    private function normalizeExpiry(int $expiry): int
-    {
-        if ($expiry && $expiry > static::THIRTY_DAYS_IN_SECONDS) {
```

Comment:
the const can be removed from the class declaration, isn't it?

Reply:
Yes, I do it

Reply:
This problem with `TestExpiration` only fails with couchbase `sdk 3.0.4` in `sdk 3.0.5` is not given. I think limiting compatibility in `sdk> = 3.0.5` is the correct solution